packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -3439,20 +5274,94 @@ export class Vector4 {
         return this;
     }
 
+    /**
+     * Updates the current Vector4 with the minimal coordinate values between its and the given coordinates
+     * @param x defines the x coordinate of the operand
+     * @param y defines the y coordinate of the operand
+     * @param z defines the z coordinate of the operand
+     * @param w defines the w coordinate of the operand
+     * @returns the current updated Vector4
+     */
+    public minimizeInPlaceFromFloats(x: number, y: number, z: number, w: number): this {
+        if (x < this.x) {
+            this.x = x;
+        }
+        if (y < this.y) {
+            this.y = y;
+        }
+        if (z < this.z) {
+            this.z = z;
+        }
+        if (w < this.w) {
+            this.w = w;
+        }
+        return this;
+    }
+
+    /**
+     * Updates the current Vector4 with the maximal coordinate values between its and the given coordinates.
+     * @param x defines the x coordinate of the operand
+     * @param y defines the y coordinate of the operand
+     * @param z defines the z coordinate of the operand
+     * @param w defines the w coordinate of the operand
+     * @returns the current updated Vector4
+     */
+    public maximizeInPlaceFromFloats(x: number, y: number, z: number, w: number): this {
+        if (x > this.x) {

```

Comment:
same than vector 3 use max

Reply:
Fixed in [157fc18](https://github.com/BabylonJS/Babylon.js/pull/13699/commits/157fc18cd530c9949d920475cb52ee5670b09539)